g2o/benchmarks/jacobian_timing_tests.cpp

```diff
@@ -0,0 +1,412 @@
+#include <benchmark/benchmark.h>
+#include <Eigen/Core>
+
+// Test several different ways of evaluating Jacobians to see the impact of different ways of implementing stuff.
+
+// 16 byte aligned allocation functions
```

Comment:
Can we here not just inlcude g2o/core/dynamic_aligned_buffer.hpp

Reply:
Good point. I know I couldn't use it at the time a while ago (I was looking at this in 2018!) but I'll see if I can remind myself why. I might have been doing some more benchmarking.

Reply:
Fixed.